.travis.yml

```diff
@@ -54,7 +54,7 @@ script:
     # Deploy PR to staging environment (only when Travis secrets are available).
     # Note: Done here (in 'script', not 'deploy') because we need deploy to happen before staging webdriver test.
     if [ ${DEPLOY_STAGING} == true ]; then
-      if [ "${TRAVIS_BRANCH}" == "master" ]; then
+      if [ "${TRAVIS_PULL_REQUEST}" == "false" -a "${TRAVIS_BRANCH}" == "master" ]; then
```

Comment:
@lukebjerring I just made one more change here. `TRAVIS_BRANCH` has different meanings in push & PR builds. In PR, it means the *base* branch of the PR, which is usually `master`, so the old condition would match both cases. I added a check `TRAVIS_PULL_REQUEST` so that it only matches master pushes.

https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables